build.gradle

```diff
@@ -287,6 +287,11 @@ subprojects {
       exceptionFormat = testExceptionFormat
     }
     logTestStdout.rehydrate(delegate, owner, this)()
+    failFast = true
```

Comment:
is there an upstream bug for this?

Reply:
1. There are a bunch failures in the security related tests, and I'm excluding them below.
It seems they only fail on our dev machines, and not on the github CI machines. And for now, no one wants to invest enough time to fix them.
2. the failFast setting means stop the testing when the first failure happens, which has always worked more nicely for me.